jsapp/scss/components/_kobo.modal.scss

```diff
@@ -865,3 +865,117 @@ $z-modal-x: 10;
     }
   }
 }
+
+.connect-projects {
+  margin: 24px;
+
+  label {
```

Comment:
This is dangerously vague, as it will apply to all `labels` inside `connect-projects`. Better make this `.connect-projects__label`